.github/workflows/CI.yml

```diff
@@ -209,7 +215,7 @@ jobs:
 
       - name: Build Amaru
         run: |
-          cargo test --release --no-run -p amaru
+          cargo test --no-run -p amaru
```

Comment:
I made the change to use `release` profile some time ago as I expect to use this build to track performance regression during `snapshots` tests.
Now we can always revisite later once we figure out how to do this.